src/main/java/net/revelc/code/impsort/maven/plugin/AbstractImpSortMojo.java

```diff
@@ -236,6 +236,20 @@ abstract class AbstractImpSortMojo extends AbstractMojo {
   @Parameter(defaultValue = "${project.build.directory}", property = "impsort.cachedir")
   private File cachedir;
 
+  /**
+   * Allows to ignore irrelevant parse errors
+   *
+   * <p>
+   * Whilst parsing all Java files completely only the parts before the first top level decleration
+   * are relevant for sorting the imports. This property allows to ignore all parse errors with a
+   * location under the first top level decleration. This allows import sorting in future Java
+   * versions that are not (yet) supported by the used Java parser.
+   *
+   * @since 1.10.0
+   */
+  @Parameter(property = "impsort.ignoreIrrelevantParseErrors", defaultValue = "false")
+  private boolean ignoreIrrelevantParseErrors;
+
```

Comment:
I don't think I'd characterize the parse errors as irrelevant. That's misleading. They are certainly relevant for the removeUnused feature. Perhaps the name should be changed to: "ignoreParseErrorsBelowImports".

Reply:
Sounds very good. ignoreParseErrorsBelowImports is much more descriptive.